schemas/blocks/definitions.json

```diff
@@ -12,7 +12,8 @@
         "$ref": "../common_definitions.json#/contents"
       }
     },
-    "additionalProperties": false
+    "additionalProperties": false,
+    "required": ["contents"]
```

Comment:
Should it not be one of `title` or `contents`?

Reply:
I think the content title should always be required? 

And it probably should be possible to have a content block with only a title?

Reply:
So I've gone with required: title now.